luigi/contrib/s3.py

```diff
@@ -309,71 +312,69 @@ def copy(self, source_path, destination_path, threads=100, start_time=None, end_
         :returns tuple (number_of_files_copied, total_size_copied_in_bytes)
         """
 
-        start = datetime.datetime.now()
-
-        (src_bucket, src_key) = self._path_to_bucket_and_key(source_path)
-        (dst_bucket, dst_key) = self._path_to_bucket_and_key(destination_path)
-
         # don't allow threads to be less than 3
         threads = 3 if threads < 3 else threads
```

Comment:
I wrote this line long ago and have always wanted to update it to `threads = max(threads, 3)`. This PR seems appropriate since you've already modified other code in this method